src/explore-education-statistics-frontend/src/modules/data-catalogue/DataSetFilePage.tsx

```diff
@@ -251,7 +253,19 @@ export default function DataSetFilePage({
                 dataSetFileId={dataSetFile.id}
                 hasApiDataSet={!!apiDataSet}
                 tableToolLink={`/data-tables/${release.publication.slug}/${release.slug}?subjectId=${file.subjectId}`}
-                onDownload={handleDownload}
+                onDownload={async () => {
```

Comment:
These chunks of code look the same. Any reason why it was changed from a single shared handleDownload function?